pkg/runtime/registry.go

```diff
@@ -106,6 +141,16 @@ func (k *KubeadmRuntime) ApplyRegistry() error {
 	if err != nil {
 		return fmt.Errorf("generator registry htpasswd failed %v", err)
 	}
+	address, err := iputils.ListLocalHostAddrs()
+	if err != nil {
+		logger.Warn("failed to get local address, %v", err)
+		address = &[]net.Addr{&net.IPNet{IP: net.ParseIP(k.getMaster0IP())}}
+	}
+	if !iputils.IsLocalIP(registry.IP, address) {
+		if err = k.copyRegistryFiles(registry); err != nil {
```

Comment:
k.copyRegistryFiles The runtime module is not suitable for placing copied files, and the copied files should be placed in the filesystem for processing.